savoirfairelinux/num2words
# adding es_HN language support

## Fixes # by frflores11

### Changes proposed in this pull request:

* Adding support for es_HN language

### Status

- [] WIP (Work-In-Progress)

### How to verify this change
print(num2words.num2words('42.32', to='currency', lang='es_HN'))
cuarenta y dos lempiras con treinta y dos centavos

*Fill out this section so that a reviewer can know how to verify your change.*

### Additional notes

*If applicable, explain the rationale behind your change.*
